ale_linters/ruby/rails_best_practices.vim

```diff
@@ -0,0 +1,55 @@
+" Author: Eddie Lebow https://github.com/elebow
+" Description: rails_best_practices, a code metric tool for rails projects
+
+let g:ale_ruby_rails_best_practices_options =
+\   get(g:, 'ale_ruby_rails_best_practices_options', '')
+
+function! ale_linters#ruby#rails_best_practices#Handle(buffer, lines) abort
+    if len(a:lines) == 0
+        return []
+    endif
+
+    let l:result = json_decode(join(a:lines, ''))
+
+    let l:output = []
+
+    for l:warning in l:result
+        if !ale#path#IsBufferPath(a:buffer, l:warning.filename)
+          continue
+        endif
+
+        call add(l:output, {
+        \    'lnum': l:warning.line_number + 0,
+        \    'type': 'W',
+        \    'text': l:warning.message,
+        \})
+    endfor
+
+    return l:output
+endfunction
+
+function! ale_linters#ruby#rails_best_practices#GetCommand(buffer) abort
+    let l:executable = ale#handlers#rails_best_practices#GetExecutable(a:buffer)
+    let l:exec_args = l:executable =~? 'bundle$'
+    \   ? ' exec rails_best_practices'
+    \   : ''
+
+    let l:rails_root = ale#ruby#FindRailsRoot(a:buffer)
+
+    if l:rails_root ==? ''
+        return ''
+    endif
+
+    return ale#Escape(l:executable) . l:exec_args
+    \    . ' --silent -f json --output-file /dev/stdout '
```

Comment:
Writing to `/dev/stdout` won't work on Windows, of course. Is there an equivalent mechanism?

If not, it may be worth an upstream PR to add a stdout option to the linter.